.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -78,6 +78,7 @@ body:
   - type: textarea
     attributes:
       label: Diagnostics information
+      render: json
```

Comment:
We want to promote users to do the right thing and drag and drop a txt file. Adding a renderer turns off the expand for file attachments:
https://gh-community.github.io/issue-template-feedback/structured/#textarea

Reply:
ah ok - was not aware of this behaviour.
Than I would suggest to highlight the part to "drag&drop, but not copy&paste the data" to the user.
Would this be ok?

Reply:
ah ok - was not aware of this behaviour.
Than I would suggest to highlight the part to "drag&drop, but not copy&paste the data" to the user.
Would this be ok?

Reply:
That is already highlighted?
![image](https://user-images.githubusercontent.com/15093472/164015681-12279733-6525-4421-b8a2-3a181c22bc9e.png)

> It would really help if you could download the diagnostics data for the device you are having issues with, and drag-and-drop that file into the textbox below.

Reply:
i mean to highlight a bit more "aggressive" - like underline the part "drag-and-drop that file into the textbox" and add something like "drag-and-drop the diagnostics data file here (do not copy-and-paste the content)" as placeholder, so it will be shown semi-transparent in the textbox

Reply:
That's ok, I think.

Reply:
underlined text is not _nativ_ supported by markdown, so will just add the `placeholder`

Reply:
`<ins>underline</ins>`

<ins>Works just fine in GitHub Markdown</ins>

Reply:
one never stops learning 🙈